hca/util/__init__.py

```diff
@@ -638,8 +640,8 @@ def build_argparse_subparsers(self, subparsers, help_menu=False):
             docstring = command.__doc__.format(prog=subparsers._prog_prefix)
             method_args = _parse_docstring(docstring)
             command_subparser = subparsers.add_parser(command.__name__.replace("_", "-"),
-                                                      help=method_args['summary'],
-                                                      description=method_args['description'],
+                                                      help=method_data.get("summary"),
+                                                      description=method_args.get('description'),
```

Comment:
It might be unhelpful to allow methods to not include `help` and a `description`.